imap_processing/idex/idex_cdf_attrs.py

```diff
@@ -0,0 +1,148 @@
+import numpy as np
+
+from .. import common_cdf_attrs
+
+# Set IDEX software version here for now
+software_version = "01"
```

Comment:
```suggestion
__version__ = "01"
```
Do we want to follow more standard Python version naming schemes here?